src/utils/migrator.rs

```diff
@@ -0,0 +1,41 @@
+use std::error::Error;
+use std::fmt::{Debug, Formatter};
+use std::sync::Arc;
+
+use sqlx::{postgres::PgPoolOptions, PgPool};
+
+const MIGRATIONS_PATH: &str = "sql/migrations";
```

Comment:
I assume this means that clippy doesn't consider code invoked from `/tests/` for this lint?

I think for now, we should opt to #[allow(lint_xyz)] to reduce these lint noises. We can remove these allows as / when they're no longer needed. WDYT?

Reply:
Yes, its strange. It right that it isn't used in the bin module, but it is exposed by the lib, and used in tests. I'll tag them for now.